# no-ember-components
Used by Ember Observer to determine if addons are using Ember components.

## Examples

Examples of **incorrect** code for this rule:

```javascript
  import Component from '@ember/component';
```
